Header header
uint32 id
float32 xPosition
float32 yPosition
floate32 zPosition
float32 xOrientation
float32 yOrientation
floate32 zOrientation

